### PR TITLE
Upload test configs

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------

When creating the ThreatExchangeConfig dynamo table in terraform, we now search for a file with configs that we should auto populate with. We do this with a new bash script that we execute from terraform whenever the table is created.

Note: terraform will only run the bash script if it is creating the table from new. To populate an existing table using the bash script you will need tot delete the table first 

Also add a config file to git that links to test configs

Test Plan
---------

Run `terraform apply` with...
- no test file variable and no config file
- test file variable and a gibberish config file
- test file variable and a valid config file

### CODE REVIEW

File: hasher-matcher-actioner/terraform/fetcher/main.tf [hunk 1]
```diff
@@ -201,7 +201,7 @@ resource "aws_iam_role_policy_attachment" "fetcher_trigger" {
 ### Config storage ###
 
 resource "aws_dynamodb_table" "threatexchange_config" {
-  name         = "${var.prefix}-ThreatExchangeConfig"
+  name         = "${var.prefix}ThreatExchangeConfig"
```
Comment (Dcallies) [inline hasher-matcher-actioner/terraform/fetcher/main.tf:204]: Why this change?

**Review by Dcallies — COMMENTED** [review]
Seems reasonable. @BarrettOlson - you ended up doing something to directly insert into the table after creation. Was this a route?

**Review by Dcallies — APPROVED** [review]
I'm fine with this, but have @schatten  or @BarrettOlson  take a quick look before landing

**Comment by jeberl** [comment]
re @Dcallies - There is a aws_dynamodb_insert command but I wasn't sure how to use it to insert an unknown known number of objects so this method seemed easier

File: hasher-matcher-actioner/terraform/fetcher/main.tf (hunk 1 above)
Comment (jeberl) [reply hasher-matcher-actioner/terraform/fetcher/main.tf:204]: `jq` was doing weird things with JSON keys that had `-` in them. COuld figure out how to escape correctly so this seemed easiest

**Review by schatten — APPROVED** [review]
Looks fine, but wouldn't this conflict with the other PR you have open #457?

**Comment by jeberl** [comment]
@schatten Will resolve conflicts

**Comment by BarrettOlson** [comment]
> Seems reasonable. @BarrettOlson - you ended up doing something to directly insert into the table after creation. Was this a route?

I used boto3 in a test script which was different as it required another command.
